locale/es_MX/LC_MESSAGES/clr-installer.po

```diff
@@ -662,6 +662,9 @@ msgstr "Kernel Linux de vanguardia adaptado a medida de metal"
 msgid "Run the latest Long Term Support (LTS) Linux Kernel"
 msgstr "Ejecute el último kernel de Linux de soporte a largo plazo (LTS)"
 
+msgid "Run the Long Term Support (LTS) 2019 Linux Kernel 5.4"
+msgstr "Ejecute el núcleo de soporte a largo plazo (LTS) 2019 Linux Kernel 5.4"
```

Comment:
Should we try to parameterize this?

msgid "Run the Long Term Support (LTS) %s Linux Kernel  %s"

Reply:
I like the idea of parametrizing strings,  in this case, it doesn't work so well as we are loading the kernel descriptions from a JSON file and then the lookup is done based on that description:
https://github.com/clearlinux/clr-installer/blob/master/gui/pages/kernel_options.go#L93